app-gateway.tf

```diff
@@ -86,7 +86,7 @@ module "appGw" {
       protocol                            = "Http"
       path                                = "/"
       interval                            = 30
```

Comment:
timeout cannot be greater than interval so I think we need to update both interval and timeout